tensorflow/python/keras/utils/np_utils.py

```diff
@@ -27,6 +27,11 @@ def to_categorical(y, num_classes=None, dtype='float32'):
 
   E.g. for use with categorical_crossentropy.
 
+  Usage Example:
+    >>> y = [0, 1, 2, 3]
+    >>> tf.keras.utils.to_categorical(y, num_classes=4)
+    array([...], dtype=float32)
```

Comment:
Please print the values here. https://www.tensorflow.org/community/contribute/docs_ref

Reply:
Sorry about that, I assumed I had to use dots here too since a reviewer in my other PR had mentioned that.